module/localisation/BallLocalisation/data/config/BallLocalisation.yaml

```diff
@@ -1,10 +1,12 @@
+# Controls the minimum log level that NUClear log will display
+log_level: DEBUG
+
 process_noise_diagonal: [0.01, 0.01]
-n_rogues: 1
+# start_state: [5, 0]
+start_variance: [0.1, 0.1]
 
-reset_range: [2, 2]
 n_particles: 30
-
-start_state: [5, 0]
-start_variance: [1, 1]
+n_rogues: 10
+reset_range: [2, 2]
```

Comment:
What does this parameter do?

Reply:
I agree, what does rogues/reset range mean?